src/apps/components/AppUrlLink/AppUrlLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { LinkIcon, Pill } from "@saleor/macaw-ui";
+import React from "react";
+
+interface AppUrlLinkProps {
+  url: string | null;
+  onClick: () => void;
+}
+
+const AppUrlLink: React.FC<AppUrlLinkProps> = ({ url, onClick }) => {
```

Comment:
I'm not sure if the name is the best possible. What about `AppPillLink` or making component more generic with name like `PillLink`?

Reply:
I think it's okay, it handles urls and has goto icon, so only improvement i see would be `UrlPill`, `PillWithUrl` or something like this. But `AppUrlLink` is okay too i guess